UI/EncyclopediaDetailPanel.cpp

```diff
@@ -1717,6 +1716,7 @@ namespace {
             if (!building_type->Effects().empty())
                 detailed_description += "\n\nEffects:\n" + Dump(building_type->Effects());
         }
+        detailed_description.append("\n");
```

Comment:
but with this extra newline:
![Bildschirmfoto 2022-01-21 um 23 16 28](https://user-images.githubusercontent.com/37298400/150607792-08e6c9c3-260a-4a01-ade5-e6899ab0d412.png)

all are now linked up. seems like the links in the pedia are not allowed to be the last thing in the page in order to be clickable

Reply:
but still. there are some things that just wont link up properly. after some more digging in the haystack, i found that some likes and dislikes refer to disabled content. e.g. the BLD_SOL_ACCEL 'Solar Accelerator' is dislyked by my fluffy pet Nymnmn and thus shows up in the pedia page but fails to be linked up because is not in the list of buildings because that content is disabled.

not clear how to proceed at this point, please advise.
a) reenable or
b) remove or comment out the reference from likes/dislikes or
c) both or something else or
d) both and something else

Reply:
I have never seen that in use. I find the idea nice (some kind of solar-forming), but probably it is safe to remove the content. Or lets start a discussion on the forums if we want something like that or not. 

@geoffthemedio you added the dislike, do you intend to have such a BLD_SOL_ACCEL building/solar-forming?

Reply:
I probably wasn't aware that it wasn't enabled, and just had it in my list of stuff I could add as likes and dislikes. So remove the reference from the likes/dislikes anywhere it appears.

Reply:
will do.

Reply:
done.